### DISCUSSION TITLE
New layout

### DISCUSSION THREAD

**Discussion by timbrownls20** [zalify/easy-email-editor#84] [Ideas]
I notice there is SimpleLayoutComponent. Could we create an alternative component with a different layout. We are interested in changing the left hand side to show content, layout and custom rather than the block tree.

I've created a mock up[ to demonstrate our idea.

![image](https://user-images.githubusercontent.com/20048601/153316658-be586ef5-3aa9-463b-8aa0-4e94831d88b4.png)

**Comment by m-Ryan** [comment]
SimpleLayout is just a simple Layout example, you can easily change it.
https://github.com/m-Ryan/easy-email-demo/pull/11/files
